### DISCUSSION TITLE
URL to Blob

### DISCUSSION THREAD

**Discussion by SplendidFarzad** [videojs/video.js#7850] [Q&A]
I need to apply the URL-to-blob practice that the hls.js performs out of the box using videojs. Basically, the [hls.js](https://github.com/video-dev/hls.js/) library, even in its simplest example turns the URL into blob using the following methods: 

```
hls.loadSource(videoSrc);
hls.attachMedia(video);
```

And this applies to live streaming m3u8 links too, which is what I need.

How can I add the blobbing functionality to a videojs instance? This is particularly difficult, because hls.js applies the src to the video element, and therefore cannot interact with video.js internal source element.

**Comment by mister-ben** [comment]
Sorry, I'm not sure what you are asking. If you inspect any player using Media Source Extensions, including Video.js and hls.js, the source of the video element will be a blob URL.
